build/evergreen_config_lib/tasks.py

```diff
@@ -28,6 +28,7 @@
 from evergreen_config_lib import shell_mongoc
 from pkg_resources import parse_version
 
+
```

Comment:
This file was formatted. I added a comment where the changes are.

Reply:
Oh nice, was this with autopep? https://pypi.org/project/autopep8/